src/replication/log_consumer.cpp

```diff
@@ -81,6 +90,7 @@ namespace cubreplication
 	      }
 
 	    delete curr_stream_entry;
+	    curr_stream_entry = NULL;
```

Comment:
What is the use for `curr_stream_entry = NULL;` ?

Reply:
The new applier_worker_task destructor needs this so as to no-op on delete